src/components/RequestModal/MovieRequestModal.tsx

```diff
@@ -199,12 +211,8 @@ const MovieRequestModal: React.FC<RequestModalProps> = ({
 
   if (editRequest) {
     const isOwner = editRequest.requestedBy.id === user?.id;
-    const showEditButton = hasPermission(
-      [Permission.MANAGE_REQUESTS, Permission.REQUEST_ADVANCED],
-      {
-        type: 'or',
-      }
-    );
+    const canManageRequests = hasPermission(Permission.MANAGE_REQUESTS);
+    const canEditRequests = hasPermission(Permission.REQUEST_ADVANCED);
```

Comment:
These are unnecessary as well.

Reply:
I can use `hasPermission` from the hook itself instead of keeping a `const` for it in the Tv Modal, but since I'm using it in quite a few places in the Movie Modal, I think that it would crowd the code too much to not define the permissions as `const`s.
Is there another way I achieve the same thing but cleaner?

Reply:
I think using `hasPermission` is fine.  It's more clear IMO.

Reply:
Gotcha